integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -140,6 +140,7 @@ <h3 class="heading font-bold text-black">
                     <div class="flex flex-col flex-auto px-4 pb-4 divide-y divide-gray-200 space-y-2">
                         <div>
                             <label>{% trans 'Address' %}</label>
+                            <div id="nominatim-error" class="bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 my-2 hidden" role="alert"></div>
```

Comment:
Now that also the address is completed, I moved the error div up to the address field